examples/jax/debugging/simple_spread/feedforward/decentralised/run_ippo.py

```diff
@@ -64,7 +64,7 @@ def main(_: Any) -> None:
     # Networks.
     def network_factory(*args: Any, **kwargs: Any) -> Any:
         return ippo.make_default_networks(  # type: ignore
-            policy_layer_sizes=(254, 254, 254),
```

Comment:
Out of interest, would be good to know why this was set to54. Obviously we want a power of 2 but weird that it was set to this

Reply:
I think it was just an arbitrary choice :)